src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -49,11 +49,17 @@ enum class IntrinsicFunctions : int64_t {
     Abs,
 
     ListIndex,
+    Partition,
     // ...
 };
 
 namespace UnaryIntrinsicFunction {
 
+#define create_var_expr(var_expr, name, intent, abi, value_attr, symtab, type)\
+    ASR::expr_t *var_expr;                                                      \
+    { create_variable(sym, name, intent, abi, value_attr, symtab, type)         \
+    var_expr = EXPR(ASR::make_Var_t(al, loc, sym)); }
+
```

Comment:
Actually we don't need this now. Please update `create_variable` macro and its usage in LPython.

https://github.com/lfortran/lfortran/blob/6eeb126cde6dcc4a1e73371e0fa3c7a010324b7f/src/libasr/pass/intrinsic_function_registry.h#L120-L126